### PR TITLE
Switch Duplicati to the image by linuxserver.json

### LINKED ISSUE #206: Unmaintained and outdated duplicati?
So i have had some issues with the duplicati contianer and went to check on it on [docker hub](https://hub.docker.com/r/intersoftlab/duplicati/dockerfile) and [github](https://github.com/dmitryint/docker-duplicati) and it looks pretty dead.

maybee you should consider replacing it with one from linuxserver.io

### PR DESCRIPTION
**Pull request by FroggyFlox**
Fixes #206.
`
### General information on project
This pull request proposes to add a new rock-on for the following project:
- name: Duplicati
- website: https://duplicati.com
- description: Free backup solution compatible with a wide-range of services.

### Information on docker image
- docker image: https://hub.docker.com/r/linuxserver/duplicati
- is an official docker image available for this project?: Yes, but this PR proposes to use the image from Linuxserver.io as it provides better management of user permissions as well as an easy path for migration from our current Duplicati image. See #206 for details.

### Checklist
- [x] Passes [JSONlint](https://jsonlint.com) validation
- [x] Entry added to `root.json` in _alphabetical_ order (for new rock-on only)
- [x] `"description"` object lists and links to the docker image used
- [x] `"description"` object provides information on the image's particularities (advantage over another existing rock-on for the same project, for instance)
- [x] `"website"` object links to project's main website

### CODE REVIEW

File: root.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "Dropbox": "dropbox.json",
     "DuckDNS": "duckdns.json",
     "Duplicati2-canary": "duplicati2-canary.json",
```
Comment (phillxnet) [inline root.json]: @FroggyFlox 
Shouldn't we be deleting this line, given we are also removing it's associated json file?
```suggestion
```

**Comment by FroggyFlox** [comment]
Thanks for catching that, @phillxnet. It was my intent to remove it but I somehow forgot it... my apologies. I committed your suggestion so it should be fine now.

**Comment by phillxnet** [comment]
@FroggyFlox Thanks for that.

I should be able to get this published shortly; just got to get done with my current task and I'll give it a final test and publish their after.

Thanks again for sorting this out. Nice to have yet another linuxserver.io offering. I'll comment here once it's published.

**Comment by phillxnet** [comment]
@FroggyFlox Just done final pre-merge test and all looks to be good.

Apologies for the delay here. Just about to merge and should be publishing shortly their after.

**Comment by phillxnet** [comment]
@FroggyFlox OK, now published. Miles better than our last offering. Well done.

I particularly like the info section.  I did manages to break my test instance but given it's in beta that may not be surprising. We can keep an eye on this and once available move this to a stable variant.

Even after wiping all config I ended up, after some abuse re turning off mid config mind, with no Web-UI for this Rock-on. Uninstall and re-install still no joy and had to in the end do a:
```
/opt/rockstor/bin/delete-rockon linuxserver/duplicati
```
Then re-install. So don't know what happened their. Do have some logs but let see how regular use pans out as I'm not a current user of Duplicati so my partial config ON/OFF with additional shares added may just have pushed the beta status over the edge.

All the same it's great to have this updated. Thanks for this and to @HBDK in the linked issue for their input on and raising the indicated issue.

**Comment by FroggyFlox** [comment]
Thanks a lot, @phillxnet!



> Even after wiping all config I ended up, after some abuse re turning off mid config mind, with no Web-UI for this Rock-on. Uninstall and re-install still no joy and had to in the end do a:
> 
> ```
> /opt/rockstor/bin/delete-rockon linuxserver/duplicati
> ```
> 
> Then re-install.

That is curious indeed... That sounds like some config remained even though you wiped all volumes... another incentive for me to tackle a user-friendly way to do this kind of "maintenance" from Rockstor's webUI once the other, more pressing, issues are behind us.

**Pull request merged**